### PR TITLE
Second catch all PR for the tutorial

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #21745

~~will be used to patch the material for day 2 & 3~~

### CODE REVIEW

**Review by loganharbour — APPROVED** [review]

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_multiphysics.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 - Scientists are adept at creating applications in their domain
 - What about collaborating across research groups and/or disciplines?
 
-  - Head in the sand?
+  - Iterating between design teams?
```
Comment (lindsayad) [inline tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_multiphysics.md:9]: This isn't clear to me. This sounds more like something that would happen at a tech company

File: tutorials/darcy_thermo_mech/doc/content/workshop/problem/step01.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ A basic MOOSE input file requires six parts, each of which will be covered in gr
 - `[Kernels]`: Define the equation(s) to solve
 - `[BCs]`: Define the boundary condition(s) of the problem
 - `[Executioner]`: Define how the problem will be solved
-- `[Outputs]`: Define how the solution will be written
+- `[Outputs]`: Define how the solution will be returned
```
Comment (lindsayad) [inline tutorials/darcy_thermo_mech/doc/content/workshop/problem/step01.md:40]: I honestly like `written` better. `returned` seems more in-code API like

**Review by lindsayad — APPROVED** [review]

File: tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_multiphysics.md (hunk 1 above)
Comment (GiudGiud) [reply tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_multiphysics.md:9]: Head in the sand was kind of sassy. People were finding their way around it, by iterating between the teams for TH, neutronics, fuel etc until everyone was happy enough

File: tutorials/darcy_thermo_mech/doc/content/workshop/problem/step01.md (hunk 2 above)
Comment (GiudGiud) [reply tutorials/darcy_thermo_mech/doc/content/workshop/problem/step01.md:40]: We ll switch it back next time then

**Pull request merged**